elasticapm/handlers/logging.py

```diff
@@ -36,9 +36,11 @@
 import traceback
 
 from elasticapm.base import Client
+from elasticapm.traces import execution_context
 from elasticapm.utils import compat
 from elasticapm.utils.encoding import to_unicode
 from elasticapm.utils.stacks import iter_stack_frames
+from utils import wrapt
```

Comment:
```suggestion
from elasticapm.utils import wrapt
```

PyCharm tends to import things without the base module, depending how you set it up (I think it has something to do with the working directory, but not sure)

Reply:
Grrr. Should have checked it when I let pycharm import it for me.